python/cuml/linear_model/elastic_net.pyx

```diff
@@ -238,3 +238,8 @@ class ElasticNet(Base, RegressorMixin):
             "tol",
             "selection",
         ]
+
+    def _more_tags():
+        return {
+            'preferred_input_order': 'F'
```

Comment:
Note: I think `F` probably should be our default, but for discussion of the adoption of the `_get_tags` API I left it as `None` to see how the changes will look

Reply:
After reviewing more estimators, I'm actually not sure `F` should be the default, we have quite a few that take `C` order

Reply:
I'd leave it as None and force estimators to opt in to specifying it.